lib/addCesiumRTC.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+var Cesium = require('cesium');
+var Cartesian3 = Cesium.Cartesian3;
+var Ellipsoid = Cesium.Ellipsoid;
+var defaultValue = Cesium.defaultValue;
+var defined = Cesium.defined;
+
+var addExtensionsUsed = require('./addExtensionsUsed');
+
+module.exports = addCesiumRTC;
+
+/**
+ * Positions the model relative to a center point using the CESIUM_RTC extension.
+ * Either specify options.position directly or calculate it from longitude, latitude and height
+ * along with an ellipsoid (WGS84 if not specified).
+ *
+ * @param {Object} gltf A javascript object holding a glTF hierarchy.
+ * @param {Object} [options] Defines the relationship with the center position
+ * @param {Cartesian3} [options.position] Places the model relative to a particular position.
+ * @param {Number} [options.longitude] The longitude to map onto the ellipsoid.
+ * @param {Number} [options.latitude] The latitude to map onto the ellipsoid.
+ * @param {Number} [options.height] Distance from the ellipsoid's surface.
+ * @param {Ellipsoid} [options.ellipsoid=Ellipsoid.WGS84] The ellipsoid to use for lat-long mapping.
+ */
+function addCesiumRTC(gltf, options) {
+    if (defined(options)) {
+        var positionArray = [];
+        var ellipsoid = defaultValue(options.ellipsoid, Ellipsoid.WGS84);
+        var position = defaultValue(options.position,
+            Cartesian3.fromRadians(options.longitude, options.latitude, options.height, ellipsoid));
```

Comment:
If `options.position` is defined it will still call `Cartesian3.fromRadians` but with undefined values.